### ISSUE TITLE
Crash 0x0053E21F 

### ISSUE THREAD

**Issue by JuniorDjjr** [gennariarmando/v-hud#4]
After my game loads, before the gameplay appears, it crashes, I still don't know which mod is causing it (if I disable all mods in my modloader it causes other crashes). If you don't know how to solve it another day I can take a look deeper. I'm using several mods, but about graphical mods I'm using only SkyGfx basically, but it wasn't caused by it.  

```
GTA SA 1.0.0.0 US
Unhandled exception at 0x0053E21F in gta_sa.exe (+0x13e21f): 0xC0000005: Access violation writing location 0x016E6850.
    Register dump:
        EAX: 0x00B73428  EBX: 0x01851084  ECX: 0x00B7D080  EDX: 0xB8E87BAD  
        EDI: 0x01850A70  ESI: 0x01850A70  EBP: 0xAEE8FFF1  EIP: 0x0053E21F  
        ESP: 0x0177F440  EFL: 0x00010207  CS: 0x00000023   SS: 0x0000002B   
        GS: 0x0000002B   FS: 0x00000053   ES: 0x0000002B   DS: 0x0000002B   
        
    Stack dump:
        0x0177F440:  01851084 6CF38C61 0177F498 0177F4FC 01851080 0177F430
        0x0177F458:  00000088 00000000 0177F4D0 7927174D 00000000 0177F4B0
        0x0177F470:  7922901F 792770C4 0184F300 00000209 1A0A1A60 0177F4E8
        0x0177F488:  79270C6D 00000000 0177F4D0 7920A605 792770E0 0184F630
        0x0177F4A0:  00000000 0184F300 0177F608 0177F604 0177F4DC 79227E93
        0x0177F4B8:  0177F4E4 0177F498 FFFFFFFF 01851070 0184F630 01851080
        0x0177F4D0:  0177F4F4 79209F5B 0177F4FC 6CF38C05 FFFFFFFF 01851070
        0x0177F4E8:  0177F660 79270C9D 00000000 0177F66C 792293C2 792770E0
        0x0177F500:  0184F630 0177F704 00000000 0177F660 792719B0 FFFFFFFF
        0x0177F518:  0177F66C 7922934A 00000000 6CF38E9D 0184F810 0177F704
        base: 0x01580000   top: 0x0177F440   bottom: 0x01780000
        
    Backtrace (may be wrong):
        =>0x0053E21F _Z13RenderEffectsv+0xaf in gta_sa.exe (+0x13e21f) (0x0177F46C) 
          0x7922901F CFontNew::Shutdown+0x227cf in VHud.asi (+0x3901f) (0x0177F4B0) 
          0x79227E93 CFontNew::Shutdown+0x21643 in VHud.asi (+0x37e93) (0x0177F4F4) 
          0x792293C2 CFontNew::Shutdown+0x22b72 in VHud.asi (+0x393c2) (0x0177F518) 
          0x7922934A CFontNew::Shutdown+0x22afa in VHud.asi (+0x3934a) (0x0177F66C) 
          0x0053EAD8 _Z4IdlePv+0x1b8 in gta_sa.exe (+0x13ead8) (0x0177F684) 
          0x77D0B8E9 in modloader.asi (+0x1b8e9) (0x0177F6CC) 
          0x77D0FC02 in modloader.asi (+0x1fc02) (0x0177F6E0) 
          0x77D0E267 in modloader.asi (+0x1e267) (0x0177F6FC) 
          0x77D10246 in modloader.asi (+0x20246) (0x0177F870) 
          0x77D1013C in modloader.asi (+0x2013c) (0x0177F87C) 
          0x0053ECC2 AppEventHandler+0xb2 in gta_sa.exe (+0x13ecc2) (0x0177F884) 
          0x00619B71 _RsEventHandler+0x11 in gta_sa.exe (+0x219b71) (0x0177F898) 
          0x00748DA0 _WinMain+0x690 in gta_sa.exe (+0x348da0) (0x0177F938) 
          0x77D0FB94 in modloader.asi (+0x1fb94) (0x0177F958) 
          0x77D0DE46 in modloader.asi (+0x1de46) (0x0177F988) 
          0x77D0B83B in modloader.asi (+0x1b83b) (0x0177F9DC) 
          0x77D0FBBB in modloader.asi (+0x1fbbb) (0x0177F9FC) 
          0x77D0E202 in modloader.asi (+0x1e202) (0x0177FA2C)

**Comment by JuniorDjjr** [comment]
I still can't make it work, I tested with only:

- ModLoader
- SilentPatch
- Widescreen Fix 2018 ([because 2020 breaks all scm interface mods](https://lh3.googleusercontent.com/-RgO3uPH0xHU/YLE7Cfn0YYI/AAAAAAAAbn4/ELFD-CkfaXkFMZ77ky2FmyRpqUyWeXGlQCLcBGAsYHQ/s16000/gta-sa-mod-widescreen-fix-bugs-problems-error.jpg))
- CLEO
- CLEO+
- NoDEP
- Framerate Vigilante

Same crash. So I started removing each mod, removing Widescreen Fix, CLEO+... Until a moment my game was only SilentPatch and CLEO, and then, tested without any mod, only ModLoader and Ginput (since this mod is not working without Ginput installed), and the same crash continues. V Hud is installed outside modloader.

The error happened right at gameplay start, after loading, I can't see anything and then crashes, the menu works (if I remove GInput, another menu crash happens before menu, but I think it's expected). I accidentally started the game with a CLEO that starts a mission, and so the crash didn't occur, but when the cutscene ended, when the gameplay appeared, crash. So I tried disabling the hud and radar in the menu and tried the gameplay again, but it didn't solve the problem.
